kofta/src/vscode-webview/components/KeybindListener.tsx

```diff
@@ -23,6 +23,17 @@ export const KeybindListener: React.FC<KeybindListenerProps> = ({}) => {
             });
             set({ muted: !muted });
           },
+          PTT: (e) => {
+            if (!e) return
+            
+            const { muted, set } = useMuteStore.getState();
+            const mute = e.type === 'keyup' 
+            wsend({
+              op: "mute",
+              d: { value: mute },
+            });
+            set({ muted: mute });
+          }
```

Comment:
don't you need 2 handlers? 1 for keyup and 1 for keydown?

Reply:
@benawad no it can be done in the same handler, we set the mute states based on if the key was pressed down or released.

Reply:
![qEnyfKRnYB](https://user-images.githubusercontent.com/38838675/107881075-5a45ac80-6f04-11eb-801a-b9382f604ab2.gif)